src/App.tsx

```diff
@@ -2,24 +1,48 @@
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const allItems = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [amounts, setAmount] = useState<number>(5);
+  const [pages, setPage] = useState<number>(1);
```

Comment:
Consider renaming the state variable `pages` to `currentPage` for better clarity, as it represents the current page number.